jcl/src/java.base/share/classes/java/lang/invoke/MethodHandleResolver.java

```diff
@@ -326,27 +400,47 @@ private static final MethodHandle sendResolveMethodHandle(
 				break;
 			case 5: /* invokeVirtual */
 				type = MethodTypeHelper.vmResolveFromMethodDescriptorString(typeDescriptor, loader, null);
+/*[IF OPENJDK_METHODHANDLES]*/
+				/* TODO: Investigate if an equivalent for lookup.accessCheckArgRetTypes is needed for OpenJDK MH. */
+/*[ELSE] OPENJDK_METHODHANDLES*/
 				lookup.accessCheckArgRetTypes(type);
+/*[ENDIF] OPENJDK_METHODHANDLES*/
 				result = lookup.findVirtual(referenceClazz, name, type);
 				break;
 			case 6: /* invokeStatic */
 				type = MethodTypeHelper.vmResolveFromMethodDescriptorString(typeDescriptor, loader, null);
+/*[IF OPENJDK_METHODHANDLES]*/
+				/* TODO: Investigate if an equivalent for lookup.accessCheckArgRetTypes is needed for OpenJDK MH. */
```

Comment:
Thanks for updating this comment - it's much clearer.  Is there an issue to track addressing this?

Reply:
Created https://github.com/eclipse/openj9/issues/10913 to track all **TODOs** for the `MethodHandleResolver` class.